include/directx/d3d12shader.h

```diff
@@ -208,54 +260,95 @@ typedef struct _D3D12_SHADER_INPUT_BIND_DESC
 #define D3D_SHADER_REQUIRES_SAMPLE_CMP_GRADIENT_OR_BIAS                                     0x80000000
 #define D3D_SHADER_REQUIRES_EXTENDED_COMMAND_INFO                                           0x100000000ull
 
-
 typedef struct _D3D12_LIBRARY_DESC
 {
     LPCSTR    Creator;           // The name of the originator of the library.
     UINT      Flags;             // Compilation flags.
     UINT      FunctionCount;     // Number of functions exported from the library.
 } D3D12_LIBRARY_DESC;
 
+typedef enum D3D12_NODE_OVERRIDES_TYPE {
+    D3D12_NODE_OVERRIDES_TYPE_NONE = 0,
+    D3D12_NODE_OVERRIDES_TYPE_BROADCASTING_LAUNCH = 1,
+    D3D12_NODE_OVERRIDES_TYPE_COALESCING_LAUNCH = 2,
+    D3D12_NODE_OVERRIDES_TYPE_THREAD_LAUNCH = 3,
+    D3D12_NODE_OVERRIDES_TYPE_COMMON_COMPUTE = 4
+} D3D12_NODE_OVERRIDES_TYPE;
+
+typedef struct _D3D12_NODE_SHADER_DESC {
+    D3D12_COMPUTE_SHADER_DESC               ComputeDesc;                  // Node extends ComputeDesc
+    D3D12_NODE_OVERRIDES_TYPE               LaunchType;                   // Launch type
+    BOOL                                    IsProgramEntry;               // Is program entry
+    INT                                     LocalRootArgumentsTableIndex; // Local root arguments table index
+    UINT                                    DispatchGrid[3];              // Dispatch grid
+    UINT                                    MaxDispatchGrid[3];           // Max dispatch grid
+    UINT                                    MaxRecursionDepth;            // Max recursion depth
+} D3D12_NODE_SHADER_DESC;
+
+typedef struct _D3D12_RAYTRACING_SHADER_DESC {
+    UINT                                    ParamPayloadSize;             // Payload or param size (callable) in bytes
+    UINT                                    AttributeSize;                // Attribute size in bytes
+} D3D12_RAYTRACING_SHADER_DESC;
+
 typedef struct _D3D12_FUNCTION_DESC
 {
-    UINT                    Version;                     // Shader version
-    LPCSTR                  Creator;                     // Creator string
-    UINT                    Flags;                       // Shader compilation/parse flags
-    
-    UINT                    ConstantBuffers;             // Number of constant buffers
-    UINT                    BoundResources;              // Number of bound resources
+    UINT                                    Version;                      // Shader version

```

Comment:
It looks like maybe there's no actual changes to D3D12_FUNCTION_DESC?  If that's the case, please revert these changes so that we can focus on reviewing the actual changes proposed.